### DISCUSSION TITLE
what is the difference between user and project repositories?

### DISCUSSION THREAD

**Discussion by Neocrumm** [community/community#3228] [Other Feature Feedback, Questions, & Ideas]
just answer

**Comment by PranamBhat** [comment]
Hi @Neocrumm 

1) A **repository** contains all of the project files (including documentation), and stores each file's revision history. Repositories can have multiple collaborators and can be either public or private. A **Project** as documented on **GitHub**: Project boards on GitHub help you organize and prioritize your work.

In general, A GitHub repository is just a "directory" where folders and files can exist.

A Repository as documented on **GitHub**:

A [**Repository**](https://docs.github.com/en/github/getting-started-with-github/github-glossary#repository) is the most basic element of **GitHub**. They're easiest to imagine as a project's folder. A repository contains all of the project files (including documentation), and stores each file's revision history. Repositories can have multiple collaborators and can be either public or private.

2) **GitHub** recently introduced a new feature called [Projects](https://github.blog/2016-09-14-a-whole-new-github-universe-announcing-new-tools-forums-and-features/#manage-your-ideas-with-projects). This provides a visual board that is typical of many Project Management tools.

A [**Project**](https://docs.github.com/en/github/managing-your-work-on-github/tracking-the-progress-of-your-work-with-project-boards) as documented on GitHub:

Project boards on GitHub help you organize and prioritize your work. You can create project boards for specific feature work, comprehensive roadmaps, or even release checklists. With project boards, you have the flexibility to create customized workflows that suit your needs.


For more information:

[https://developer.github.com/v3/repos/](https://developer.github.com/v3/repos/)

[https://developer.github.com/v3/projects/](https://developer.github.com/v3/projects/)

**Reply by Gra-ach** [reply]
Hello @PranamBhat The question was not about a difference between repository and project but between "repository project" and "user project". Unfortunately, your suggestion doesn't answer the question. And I would really like to know this difference as well.

**Reply by AllisonGarner** [reply]
I believe that the user repository is one where you can keep your personal files/projects, its like a private repository, whereas the project repository is more like a shared space having people working on the same project as you.

**Reply by calvinc460** [reply]
@Gra-ach, @AllisonGarner - I'm sort new to GitHub myself, so my understanding may not be complete, but as far as I know, GitHub has repositories - they are not broken down into user repositories vs project repositories.  A repository can be private or public, and it can be owned by an individual user or by an organization (which is probably - but not necessarily - a group of people working on projects, as well as on several repositories).  

Separately, there are projects, which may refer to repositories.  Main word here - separately.  A project is not a repository nor a type of repository (and vice versa), although there can be interaction/references between projects and repositories.

Past this, I refer you to some of the other answers to this question.

**Comment by slabua** [comment]
Types of project boards:

- User-owned classic project can contain issues and pull requests from any personal repository.

- Organization-wide classic project can contain issues and pull requests from any repository that belongs to an organization. You can link up to twenty-five repositories to your organization or user-owned classic project. Linking repositories makes it easier to add issues and pull requests from those repositories to your classic project using  Add cards or from the issue or pull requests sidebar. For more information, see "[Linking a repository to a project (classic)](https://docs.github.com/en/articles/linking-a-repository-to-a-project-board)."

- Repository classic project are scoped to issues and pull requests within a single repository. They can also include notes that reference issues and pull requests in other repositories.

---
[Documentation](https://docs.github.com/en/issues/organizing-your-work-with-project-boards/managing-project-boards/about-project-boards)

**Comment by rdyar** [comment]
Maybe this has changed but there used to be a difference between a user repo and a project repo from a jekyll standpoint - the user repo was named `myusername.github.io` and if you had one of those then GH Pages served that site as you normally would where the files to serve were all in the root of the site. All other repos in your account are served from sub folders of your username and so you have to use a baseurl setting to prepend the project/repo name to all the urls.

So it is sort of a directory structure on their end where your user repo with its special name could skip the baseurl but any other repo that you want to serve via GH pages needed to use the repo name in the path to all the files as the GH web server was really serving the root of the directory. Something like that. I still have a user repo and it works this way still though I can't seem to find much about it other than this discussion.
